### PR TITLE
Install CRD's in local cluster

### LINKED ISSUE #70791: local-up-cluster.sh: install CSI CRDs
**What would you like to be added**:

https://github.com/kubernetes/kubernetes/pull/70193/ added a new addon to addon manager "that automatically installs CSI CRDs if CSIDriverRegistry or CSINodeInfo feature gates are true". But this is either broken or this addon manager isn't active in a cluster brough up with:
```
ALLOW_PRIVILEGED=1 FEATURE_GATES="CSIPersistentVolume=true,MountPropagation=true,VolumeSnapshotDataSource=true,KubeletPluginsWatcher=true,CSINodeInfo=true,CSIDriverRegistry=true"  hack/local-up-cluster.sh -O
```

This should work automatically also for such local clusters.

**Why is this needed**:

The CSI E2E tests are now expecting that the cluster has the CRDs and fail when that isn't the case.

/kind feature
(or perhaps a bug, not sure)

/cc @saad-ali

### PR DESCRIPTION
**Pull request by xing-yang**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
This PR installs CSIDriver and CSINodeInfo CRD's in the local cluster.

**Which issue(s) this PR fixes**:

Fixes #70791

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
Install CSINodeInfo and CSIDriver CRDs in the local cluster.
```

### CODE REVIEW

**Comment by xing-yang** [comment]
@msau42 @saad-ali

**Comment by msau42** [comment]
/lgtm cancel
@xing-yang the failure looks like it might be an issue caused by the PR:
```
W0107 23:56:11.022] /go/src/k8s.io/kubernetes/hack/local-up-cluster.sh: line 868: ps: command not found
```

File: hack/local-up-cluster.sh [hunk 1]
```diff
@@ -968,6 +968,28 @@ function create_storage_class {
...
+    YAML_FILE=${KUBE_ROOT}/cluster/addons/storage-crds/csinodeinfo.yaml
+
+    if [ -e $YAML_FILE ]; then
+        echo "Create CSINodeInfo CRD"
+        ${KUBECTL} --kubeconfig="${CERT_DIR}/admin.kubeconfig" create -f $YAML_FILE
+    else
+        echo "No csinodeinfo available."
+    fi
+}
+
```
Comment (bertinatto) [inline hack/local-up-cluster.sh]: Alternative: we could unify this into a `create_crd` function that takes a YAML file as parameter.

File: hack/local-up-cluster.sh [hunk 2]
```diff
@@ -1112,6 +1134,14 @@ if [[ "$DEFAULT_STORAGE_CLASS" = "true" ]]; then
...
 fi
 
+if [[ ${FEATURE_GATES} == *"CSIDriverRegistry=true"* ]]; then
+  create_csidriver
+fi
+
+if [[ ${FEATURE_GATES} == *"CSINodeInfo=true"* ]]; then
+  create_csinodeinfo
+fi
+
```
Comment (bertinatto) [inline hack/local-up-cluster.sh]: I think it wouldn't install the CRDs if `FEATURE_GATES` is set to `AllAlpha=true`, right?

Comment (bertinatto) [reply hack/local-up-cluster.sh]: Would it be better to do something like this instead?

```if [[ "${FEATURE_GATES:-}" =~ "CSIDriverRegistry=true" || (...)```

File: hack/local-up-cluster.sh (hunk 1 above)
Comment (xing-yang) [reply hack/local-up-cluster.sh]: Sure

File: hack/local-up-cluster.sh (hunk 2 above)
Comment (xing-yang) [reply hack/local-up-cluster.sh]: Yes, I'll change it.  Thanks!

**Comment by bertinatto** [comment]
Interesting, occasionally `pkill` won't be found in `$PATH`:

```
201 | W0109 09:35:35.834] 2019/01/09 09:35:35 process.go:153: Running: pkill hyperkube
202 | W0109 09:35:35.884] 2019/01/09 09:35:35 process.go:155: Step 'pkill hyperkube' finished in 110.094µs
203 | W0109 09:35:35.884] 2019/01/09 09:35:35 local.go:225: unable to kill hyperkube processes: error starting pkill hyperkube: exec: "pkill": executable file not found in $PATH
204 | W0109 09:35:35.884] 2019/01/09 09:35:35 process.go:153: Running: pkill etcd
205 | W0109 09:35:35.885] 2019/01/09 09:35:35 process.go:155: Step 'pkill etcd' finished in 8.562µs
```

**Pull request merged**